src/tim/vx/tensor.cc

```diff
@@ -388,7 +388,7 @@ TensorSpec& TensorSpec::SetQuantization(Quantization& quantization) {
 }
 
 TensorSpec TensorSpec::AsTransientSpec() const {
-  return TensorSpec(this->datatype_, ShapeType({}), TensorAttribute::TRANSIENT,
+  return TensorSpec(this->datatype_, this->shape_, TensorAttribute::TRANSIENT,
```

Comment:
Original idea for Transient Tensor is : we suppose ovxlib will put correct shape after op_setup. Why we need to give the shape explicitly here?

Reply:
Before op_setup, batchfuse in timx-vx need tensor shape to compute the fused shape. if we do not set this shape explicitly, we will get empty shape when we run model in tflite-vx-delegate